SeQuant/core/eval_expr.cpp

```diff
@@ -146,17 +146,17 @@ bool EvalExpr::is_variable() const noexcept {
   return expr().is<Variable>() && result_type() == ResultType::Scalar;
 }
 
-Tensor const& EvalExpr::as_tensor() const noexcept {
-  return expr().as<Tensor>();
-}
+bool EvalExpr::is_id() const noexcept { return op_type() == EvalOp::Id; }
```

Comment:
what's ID? value?

Reply:
The leaf evaluation is an identity evaluation type -- because it is neither a sum nor a product. Should I rename it to `is_atomic` and make `EvalOp::Id` to `EvalOp::Atom`?

Reply:
or maybe `is_atom`?